lib/linguist/languages.yml

```diff
@@ -1292,6 +1292,7 @@ F#:
   codemirror_mime_type: text/x-fsharp
   language_id: 105
 F*:
+  fs_name: Fstar
```

Comment:
We should describe this new field in `language.yml`'s header, along with the other fields. Its purpose or meaning isn't immediately obvious unless the reader knows that `fs_` stands for `filesystem_` / `file_safe_`).

Reply:
Oh, right! We'll do as soon as I get home.
`filesystem` here, I guess I'm used to seeing it in the system community.

Reply:
Done.

Reply:
Or even fstar_name in this case!